src/app/static/src/app/components/ErrorAlert.vue

```diff
@@ -26,7 +19,6 @@
 
     interface Computed {
         message: string,
-        hasTrace: boolean,
         cssClass: string
```

Comment:
`cssClass` and `showTrace` and `toggleTrace` can be removed now.

Reply:
Thanks! Tidied that up now